extensions/JeremyGamer13/tween.js

```diff
@@ -0,0 +1,311 @@
+    */
+
+    const EasingMethods = [
+        "linear",
+        "sine",
+        "quad",
+        "cubic",
+        "quart",
+        "quint",
+        "expo",
+        "circ",
+        "back",
+        "elastic",
+        "bounce"
+    ];
+
+    const BlockType = Scratch.BlockType;
+    const ArgumentType = Scratch.ArgumentType;
+    const Cast = Scratch.Cast;
+
+    class Tween {
+        getInfo() {
+            return {
+                id: 'tweening',

```

Comment:
put your username in there, eg. "jeremygamer13tweening" or "jeremygamertweening", something like that